src/main/java/net/masterthought/cucumber/json/Step.java

```diff
@@ -13,7 +15,9 @@
     private final Row[] rows = new Row[0];
     private final Match match = null;
     private final Embedding[] embeddings = new Embedding[0];
-    private final Output output = null;
+    @JsonDeserialize(using = OutputsDeserializer.class)
+    @JsonProperty("output")
+    private final Output[] outputs = new Output[0];
```

Comment:
No, I haven't tried. Because in this case we need to collect outputs (if we have more than 1 source) inside Output class in array and every element of this array should contain messages. But I can do this. What I should do?